scripts/packaging/build_darwin_package.sh

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env bash
+set -o pipefail
+
+git config --global --add safe.directory /snowflake-cli
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+PACKAGING_DIR=$ROOT_DIR/scripts/packaging
+
+SYSTEM=$(uname -s | tr '[:upper:]' '[:lower:]')
+MACHINE=$(uname -m | tr '[:upper:]' '[:lower:]')
+
+CLI_VERSION=$(hatch version)
+
+DIST_DIR=$ROOT_DIR/dist
+APP_NAME=SnowflakeCLI.app
+APP_DIR=$DIST_DIR/app
+APP_SCRIPTS=$DIST_DIR/scripts
+
+loginfo() {
+  logger -s -p INFO -- $1
+}
+
+loginfo "Building darwin package for version ${CLI_VERSION}"
+
+rm -rf $APP_DIR || true
+mkdir -p $APP_DIR || true
+cd $APP_DIR
+
+mkdir -p $APP_NAME/Contents/{MacOS,Resources}
+
+cat >$APP_NAME/Contents/Info.plist <<INFO_PLIST
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>CFBundleName</key>
+    <string>SnowflakeCLI</string>
+    <key>CFBundleDisplayName</key>
+    <string>SnowflakeCLI</string>
+    <key>CFBundleIdentifier</key>
+    <string>net.snowflake.snowflake-cli</string>
```

Comment:
`net`?

Reply:
I used snowsql files as template.
I'm did not changed those entries on purpose.

Reply:
24h hours later we have a working build with `com.snowflake`